modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryPerformanceTest.java

```diff
@@ -132,33 +136,206 @@ private void _addObjectEntries(Integer numberOfEntries) throws Exception {
 				},
 				ObjectDefinitionConstants.SCOPE_COMPANY);
 		}
+
+		try (Closeable closeable = new PerformanceTimer(
+				GetterUtil.getInteger(
+					_properties.getProperty("object.entries.get.max.time")),
+				"Get all the Object Entries by ObjectEntryLocalService with " +
+					"CustomObjectDefinitionId:" +
+						_customObjectDefinition.getObjectDefinitionId())) {
+
+			_objectEntryLocalService.getObjectEntries(
+				0, _customObjectDefinition.getObjectDefinitionId(),
+				QueryUtil.ALL_POS, QueryUtil.ALL_POS);
+		}
+	}
+
+	@Test
+	public void testHttpPostAndDeleteObjectEntries() throws Throwable {
+		TransactionConfig.Builder transactionConfigBuilder =
+			new TransactionConfig.Builder();
+
+		_company = TransactionInvokerUtil.invoke(
+			transactionConfigBuilder.setPropagation(
+				Propagation.REQUIRED
+			).setRollbackForClasses(
+				Exception.class
+			).build(),
+			() -> {
+				Company company = CompanyLocalServiceUtil.addCompany(
+					null, _VIRTUAL_HOST_NAME, _VIRTUAL_HOST_NAME,
+					_VIRTUAL_HOST_NAME, 0, true, true, null, null, null, null,
+					null, null);
+
+				PortalInstances.initCompany(company);
+
+				return company;
+			});
+
+		try {
+			_invokeHttp(null, HttpInvoker.HttpMethod.GET, _getPath(""));
+		}
+		catch (ConnectException connectException) {
+			throw new Exception(
+				StringBundler.concat(
+					"Please add the virtual host name: ", _VIRTUAL_HOST_NAME,
+					" to the hosts file and rerun the test!"),
+				connectException);
+		}
+
+		ObjectFolderResource.Builder objectFolderResourceBuilder =
+			_objectFolderResourceFactory.create();
+
+		ObjectFolderResource objectFolderResource =
+			objectFolderResourceBuilder.user(
+				UserTestUtil.getAdminUser(_company.getCompanyId())
+			).build();
+
+		objectFolderResource.setContextCompany(_company);
+
+		JSONObject objectFolderJSONObject = _jsonFactory.createJSONObject(
+			StringUtil.read(
+				ObjectEntryPerformanceTest.class.getClassLoader(),
+				"/com/liferay/object/web/internal/object/definitions/portlet" +
+					"/action/test/dependencies/test-object-folder-4.json"));
+
+		ObjectFolder objectFolder = ObjectFolder.toDTO(
+			objectFolderJSONObject.toString());
+
+		objectFolder.setName("SampleObjectFolder");
+
+		objectFolderResource.putObjectFolderByExternalReferenceCode(
+			objectFolder.getExternalReferenceCode(), objectFolder);
+
+		try (PerformanceTimer performanceTimer1 = new PerformanceTimer(
+				GetterUtil.getInteger(
+					_properties.getProperty("object.entries.import.max.time")),
+				StringBundler.concat(
+					"Import all the ", _objectEntriesCount, " Object Entries ",
+					"by HttpInvoker"))) {
```

Comment:
"by HTTP POST"

Reply:
Maybe it's better to state these object entries are from the "SampleObjectFolder" object folder. Same with DELETE.